src/main/java/RacingGame/Domain/BigNumberStrategy.java

```diff
@@ -0,0 +1,15 @@
+package RacingGame.Domain;
+
+import java.util.Random;
+
+public class BigNumberStrategy implements RandomNumberStrategy {
+    @Override
+    public int GenerateRandomNumber() {
+        Random random = new Random();
+        int ret = random.nextInt(10);
+        while (ret < 4){
+            ret = random.nextInt(10);
+        }
+        return ret;
```

Comment:
BigNumberStrategy 라면 항상 4보다 큰 숫자를 리턴하면 되지 않을까요?

```suggestion
        return 5;
```